setup-scripts/health-checker.py

```diff
@@ -211,60 +218,118 @@ async def check_health():
             client, message, file=failed_records_file, force_notify=force_notify
         )
 
+
+# contains_string is a simple helper to check if a string contains a string.
+# This is faster and easier than regex for word comparisons
+def contains_string(string_to_check, string_to_find):
+    return string_to_find in string_to_check
+
 # check_alerts checks the alerts returned from siad's daemon/alerts API
 async def check_alerts():
     print("\nChecking portal siad alerts...")
 
-    try:
-        alerts_res = requests.get("http://localhost:9980/daemon/alerts",headers={"User-Agent":"Sia-Agent"}, verify=False)
-        alerts_json = alerts_res.json()
-    except:
-        trace = traceback.format_exc()
-        print("[DEBUG] check_alerts() failed.")
-        return await send_msg(
-            client, "Failed to run the checks!", file=trace, force_notify=True
-        )
-
-    alerts =  alerts_json['alerts']
-    critical_alerts =  alerts_json['criticalalerts']
-    error_alerts =  alerts_json['erroralerts']
-    warning_alerts =  alerts_json['warningalerts']
+    # Execute siac alerts and read the response
+    # TODO: is the container name always `sia` for production servers? Is it
+    # only changed to the server name when it it is moved to Maintenance? Will
+    # this just never check the alerts on the maintenance servers?
+    cmd_string = "docker exec sia siac alert"
```

Comment:
Do we have a docker container variable name? `sia` works for the production portals, but then won't work for the maintenance servers.

I guess I'm assuming this same script is running on the maintenance servers, is that true? This is a good check to be running on both so I want to make sure we do that.

Reply:
Yes, we do. The crontab entry that's calling this script looks like this:
```
0 * * * * ~/skynet-webportal/setup-scripts/health-checker.py ~/skynet-webportal/.env sia 1
```
The first parameter is the container name.

On the maintenance server we're not running the health-check script. Maybe we should but we don't right now, so we can worry about that in a future PR.

Reply:
I added that.